QueryCiv3/BiqSections/Wchr.cs

```diff
@@ -0,0 +1,24 @@
+using System;
+using System.Runtime.InteropServices;
+
+namespace QueryCiv3.Biq
+{
+    [StructLayout(LayoutKind.Sequential, Pack=1)]
+    public unsafe struct WCHR
+    {
+        public int Length;
+        public int Climate; // 0: Arid, 1: Normal, 2: Wet, 3: Random
```

Comment:
I like the Apolyton documentation's differentiation here of "selectedClimate"/"actualClimate".  We had some confusion over "terrain"/"overlayTerrain", and now call is "baseTerrain/overlayTerrain" since when there are two variants of a variable, it's easy to confuse them without the variant name being specified.

I do like that you've included the documentation of which type is which.  I wonder if long-term we'll want enums for those?  That would probably make more sense in the C7 side than the Civ3 import side though.